### PR TITLE
Sumo-NetFlow: first checkin

### PR DESCRIPTION
**Pull request by netflowlogic-com**
# PR Details

First checkin

## Description

First version of the app

## Checklist

- [x] Changes follows the best practices from the given [docs](https://docs.google.com/document/d/1xfYfruru0RFWOH23GRrRzosSlepJUTcakLFHLHdpEtc/edit?usp=sharing).
- [ ] Updated CHANGELOG.md.
- [x] Ran unit tests locally.
- [x] Updated screenshots/sample logs if any.

### CODE REVIEW

**Comment by JKashyap96** [comment]
@netflowlogic-com Hello, please provide information regarding how to ingest/integrate netflow logic with sumo. You can refer this section as a [reference](https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/LambdaTest#collect-logs-for-lambdatest) for collecting logs for netflow logic.

File: NetFlow/NetFlow.manifest.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "parameters": [
+        {
+            "dataSourceType": "LOG",
+            "description": "LOG data source",
+            "example": null,
+            "label": "LOG data source",
+            "parameterId": "logsrc",
+            "parameterType": "DATA_SOURCE"
+        },
+        {
```
Comment (JKashyap96) [inline NetFlow/NetFlow.manifest.json:25]: There should be a single log source here.

**Comment by JKashyap96** [comment]
For this screenshot, NetFlow/resources/screenshots/sumo_security_critical_ports.png, please upload one with all panels showing data, I see a en empty panel here.

File: NetFlow/NetFlow.manifest.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+    "author": "NetFlow Logic",
+    "categories": [
+        "Compliance and Security",
+        "IT Infrastructure",
+        "Sumo Logic Certified"
+    ],
+    "communityURL": "https://support.sumologic.com/hc/en-us/community/topics/200263058-Applications-and-Integrations",
+    "description": "The NetFlow Optimizer app provides dashboards to monitor your network traffic in data centers or clouds providing details on inbound/outbound accepted rejected flows, source and destination IP addresses, ports, protocols, as well as usage by application and user. Data is imported into Sumo via JSON streaming of NetFlow Optimizer processing NetFlow, IPFIX, sFlow, and cloud VPC flow logs.",
+    "helpURL": "https://docs2.netflowlogic.com/integrations-and-apps/integration-with-sumo-logic/",
```
Comment (rushan111) [inline NetFlow/NetFlow.manifest.json:10]: Nice job. I just think correct link is:
https://docs.netflowlogic.com/integrations-and-apps/integration-with-sumo-logic/

Comment (imrago-medxnote) [reply NetFlow/NetFlow.manifest.json:10]: confirmed, it was not updated

**Comment by imrago-medxnote** [comment]
> For this screenshot, NetFlow/resources/screenshots/sumo_security_critical_ports.png, please upload one with all panels showing data, I see a en empty panel here.

This is expected. The screenshot was taken in our lab where is not inbound DNS traffic as we use Google DNS in o0ur lab, IP addresses 8.8.8.8 and 8.8.4.4

File: NetFlow/NetFlow.manifest.json (hunk 1 above)
Comment (imrago-medxnote) [reply NetFlow/NetFlow.manifest.json:25]: Fixing

**Comment by rushan111** [comment]
Hello! @imrago-medxnote @Thanks adding a line on each pipe, that's a good work! 
I wanted to discuss regarding option to send the data to the Sumologic via HTTP collect. Is it something we can implment as well? 

Thanks

**Comment by netflowlogic-com** [comment]
> Hello! @imrago-medxnote @thanks adding a line on each pipe, that's a good work! I wanted to discuss regarding option to send the data to the Sumologic via HTTP collect. Is it something we can implement as well?
> 
> Thanks

Thanks, we created an endpoint and we are preparing for the sending.

**Review by rushan111 — APPROVED** [review]
All looks good on my side! Good job!

**Pull request merged**